src/index.js

```diff
@@ -54,4 +54,45 @@ document.querySelector('#task-list').addEventListener('change', (e) => {
     // Reload page
     location.reload();
   }
+});
+
+// Event: editing and removing
+document.querySelector('#task-list').addEventListener('click', (e) => {
+  const classesIcn = e.target.className;
+  const classesArr = classesIcn.split(' ');
+
+  const li = e.target.parentElement.parentElement;
+  const ulList = document.querySelector('#task-list');
+  const nodes = Array.from(ulList.children);
+  const index = nodes.indexOf(li);
```

Comment:
- [ ] The index of first to-do item should be `1` as request in [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#project-requirements). Currently it starts from `0`.

> By default new tasks should have the property completed set to false and the  property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).